app/ui/src/app/store/step/step.store.ts

```diff
@@ -92,19 +91,52 @@ export class StepStore {
       description: "Sends a message to the integration's log",
       configuredProperties: undefined,
       properties: {
-        message: {
+        expression: {
           type: 'string',
           displayName: 'Log Message',
-          required: true,
+          required: false,
+        },
+        bodyLoggingEnabled: {
```

Comment:
@gashcrumb: This does work fine, but when I edit the checkbox always comes unchecked. Any clues?

Reply:
I would verify via the network tab in the browser what the step configuration looks like in the integration when it's fetched first and just make sure that the propery is set in `configuredProperties`

Reply:
From the [look of this](https://github.com/udos86/ng-dynamic-forms/blob/master/packages/core/src/model/dynamic-check-control.model.ts#L19) also worth verifying that the value is actually a boolean and not a string.

Reply:
I think that last time I used the network tab to inspect the value was string. Need to check again.